src/mono/sample/wasm/browser-bench/Span.cs

```diff
@@ -83,6 +84,22 @@ public override void RunStep()
             }
         }
 
+        class IndexOfString : SpanMeasurement
+        {
+            public override string Name => "IndexOf strings";
+
+            string input = "string1";
```

Comment:
It's probably appropriate to make these strings a bit larger, so that vectorization can engage. I think they're too small as-is

Reply:
I use these as I want to replicate one of microbenchmarks, where we are slow now.